src/views/v2/rsvp/form/fields/name.php

```diff
@@ -34,8 +34,7 @@ class="tribe-common-b2--min-medium tribe-tickets__form-field-label"
 	</label>
 	<input
 		type="text"
-		id="tribe-tickets-rsvp-name"
```

Comment:
doesn't the label above need this (for accessibility and to link the label to the input, such as clicking on the label)?

Reply:
You're absolutely right! When I implemented the JS I tried to clean up the templates and forgot that we had a label there. Thanks